jp/components/drop_down.md

```diff
@@ -0,0 +1,283 @@
+﻿---
+title: Drop Down コンポーネント
+_description: Ignite UI for Angular Drop Down は、単一項目の選択が可能なグループ化された項目のリストにスクロールを表示します。
+_keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Angular Drop Down components, Angular Drop Down controls
+---
```

Comment:
@yoshimis There needs to be an extra metadata field added to all new Japanese files.
`_language: ja`